docs/pages/bare/using-expo-cli.mdx

```diff
@@ -6,46 +6,57 @@ description: Learn how and why you can use Expo CLI instead of @react-native-com
 import { Terminal } from '~/ui/components/Snippet';
 import { Collapsible } from '~/ui/components/Collapsible';
 import { BoxLink } from '~/ui/components/BoxLink';
+import { DEMI } from '~/ui/components/Text';
 
-To use Expo CLI in a project created with `npx react-native init`, you will first need to install the `expo` package. This package includes the Expo Modules API and Expo CLI. For a complete explanation of how to install this, see the ["Install Expo modules" guide](/bare/installing-expo-modules.mdx). For most apps, it's as simple as running the following command in your project directory:
+To use Expo CLI in a project created with `npx react-native init`, you will first need to install the `expo` package.
+This package includes the Expo Modules API and Expo CLI. For a complete explanation of how to install this, see the ["Install Expo modules" guide](/bare/installing-expo-modules.mdx).
```

Comment:
```suggestion
This package includes the Expo Modules API and Expo CLI. For a complete explanation of how to install this, see the [Install Expo modules](/bare/installing-expo-modules.mdx) guide.
```

Removing quotes from the link for consistency. I think the link highlight on the specific term is enough in these cases rather than quotes.